.github/workflows/check.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           sudo mv parallel sem /usr/local/bin
 
       - name: Format
-        if: github.ref != 'refs/heads/master'
+        if: github.ref != 'refs/heads/next'
```

Comment:
@hydrosquall do we still need this?

Reply:
I think so, assuming the goal here is to warn someone if they tried to open a PR with some unformatted code